stable_baselines3/common/env_util.py

```diff
@@ -122,7 +122,10 @@ def _init() -> gym.Env:
         # Default: use a DummyVecEnv
         vec_env_cls = DummyVecEnv
 
-    return vec_env_cls([make_env(i + start_index) for i in range(n_envs)], **vec_env_kwargs)
+    vec_env = vec_env_cls([make_env(i + start_index) for i in range(n_envs)], **vec_env_kwargs)
+    # Prepare the seeds for the first reset
+    vec_env.seed(seed)
```

Comment:
Simple comment: in this context `vec_env.seed` affects a unique seed for each of the environments in the same manner as `env.action_space.seed(seed + rank)`